thorlcr/activities/indexread/thindexreadslave.cpp

```diff
@@ -735,11 +755,7 @@ class CIndexReadSlaveBase : public CSlaveActivity
     }
     virtual void done() override
     {
-        {
-            CriticalBlock b(ioStatsCS);
-            updateStats();
-            lazyIFileIO.clear();
-        }
+        lazyIFileIO.clear();
```

Comment:
The stats gathering has been moved to gatherActiveStats so that the stats are updated periodically whilst the activity is running.